ui/modal/modalConfirm/view.jsx

```diff
@@ -12,28 +12,21 @@ type Props = {
   body?: string | Node,
   labelOk?: string,
   labelCancel?: string,
-  onConfirm: (closeModal: () => void, setIsBusy: (boolean) => void) => void,
```

Comment:
sorry, reverting

Reply:
A potential solution:
- Allow 2 types of props:  `onConfirm` and `onConfirmAsync`.  The he latter will pass the closing work to the client.  If both are provided (user error), use the latter.

_But if want to change, let's do it in another PR since this one might be squashed, making it hard to cherry-pick back to SBL branch._

Reply:
Side-note:  
I think modals like Tipping should be doing the same, i.e. spin until API is done before closing modal (the modal can still be manually dismissed from the X button if user don't want to wait).

I always feel anxious whether the tip went through or not, as the toast takes some time to appear.